### DISCUSSION TITLE
New Source: Bling ERP System

### DISCUSSION THREAD

**Discussion by Muriloo** [airbytehq/airbyte#35258] [New Connector Request]
## Tell us about the new integration you’d like to have
Integration of [Bling ERP system](https://www.bling.com.br/home).
Bling is a product used as an ERP system for small and medium companies in Brazil.

## Describe the context around this new integration
This integration is used to get data such as orders, products, customers and invoices from the erp system in order that many internal teams can analyze this data.
The integration is done via [Bling API](https://ajuda.bling.com.br/hc/pt-br/categories/360002186394-API-para-Desenvolvedores) and it's run a few times every day.
The tap version is already created [here](https://github.com/Muriloo/tap-bling-erp).

## Describe the alternative you are considering or using
I would consider using Meltano to run this tap as an alternative.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368096829555) by [Unito](https://www.unito.io)

**Comment by NatutechIntegracao** [comment]
> ## Tell us about the new integration you’d like to have
> Integration of [Bling ERP system](https://www.bling.com.br/home). Bling is a product used as an ERP system for small and medium companies in Brazil.
> 
> ## Describe the context around this new integration
> This integration is used to get data such as orders, products, customers and invoices from the erp system in order that many internal teams can analyze this data. The integration is done via [Bling API](https://ajuda.bling.com.br/hc/pt-br/categories/360002186394-API-para-Desenvolvedores) and it's run a few times every day. The tap version is already created [here](https://github.com/Muriloo/tap-bling-erp).
> 
> ## Describe the alternative you are considering or using
> I would consider using Meltano to run this tap as an alternative.
> 
> ┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368096829555) by [Unito](https://www.unito.io)

Oi Murilo, vc trabalha com a Bling?

[a comment by an automated account is not shown]
